pkgs/development/libraries/cpp-netlib/default.nix

```diff
@@ -14,18 +14,22 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ cmake boost openssl ];
 
-  # This can be removed when updating to 0.13, see https://github.com/cpp-netlib/cpp-netlib/issues/629
-  propagatedBuildInputs = [ asio ];
-
   cmakeFlags = [
     "-DCPP-NETLIB_BUILD_SHARED_LIBS=ON"
   ];
 
   enableParallelBuilding = true;
 
+  # The test driver binary lacks an RPath to the library's libs
+  preCheck = ''
+    export LD_LIBRARY_PATH=$PWD/libs/network/src
+  '';
+
+  # Most tests make network GET requests to various websites
+  doCheck = false;
```

Comment:
All of this is a no-op change from the status quo, since `doCheck = false` is the default. However, I spent some time getting the test driver to at least execute and pass some tests (with above `preCheck` hook), and felt like it was worth documenting.  A handful of tests pass, but a significant enough majority try to do something like `GET https://github.com` that I felt like it wasn't worth going through and patching them all out.